workloads/kube-burner/run.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/bash
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+
+case ${WORKLOAD} in
+  cluster-density)
+    WORKLOAD_TEMPLATE=workloads/cluster-density/cluster-density.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics-aggregated.yaml}
+    export TEST_JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
+  ;;
+  node-density)
+    WORKLOAD_TEMPLATE=workloads/node-pod-density/node-pod-density.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+    NODE_COUNT=${NODE_COUNT:-4}
```

Comment:
@rsevilla87 Not specific to this pr as it is the current case as well but why do we default to 4 nodes? Wouldn't it make more sense to default to the current worker count?

Reply:
you're 100% right, I set it to 4 nodes due to historical reasons, transitioning it to worker node count